core/src/main/java/org/apache/accumulo/core/iterators/system/LocalityGroupIterator.java

```diff
@@ -134,7 +129,7 @@ public int getNumLGSeeked() {
 
   public LocalityGroupIterator(LocalityGroup[] groups) {
     super(groups.length);
-    this.lgContext = new LocalityGroupContext(groups);
+    this.lgContext = new LocalityGroupContext(Arrays.asList(groups));
```

Comment:
Since this system iterator is also not public API, I wonder if there's something we can do here to pass this in as a list initially, rather than as an array that we have to convert. This could also be done in subsequent work. What do you think, @belugabehr ? Is it worth going down the rabbit hole a bit further on this one, or doing in a future change?